### ISSUE TITLE
pyenv initialization in .bash_profile vs. .bashrc (after release 2.0.0)

### ISSUE THREAD

**Issue by JayLDoherty** [pyenv/pyenv#1952]
With the recent release of pyenv 2.0.0, I believe that the documentation currently recommends the following configuration for a MacOS bash shell:

```
# inside ~/.bash_profile
eval "$(pyenv init --path)"

# inside ~/.bashrc
eval "$(pyenv init -)"
```

If I'm not allowed to edit .bashrc on my system (for reasons completely unrelated to pyenv), what would be the drawbacks of just putting both inside my profile, like this:

```
# inside ~/.bash_profile
eval "$(pyenv init --path)"
eval "$(pyenv init -)"
```

I've been using it like this successfully for a few hours (on MacOS, with a Homebrew installation of pyenv, and Python 3.6/3.7/3.8 virtual environments created with venv). I'm trying to understand what might be the drawbacks of configuring my system this way, because the documentation makes a big deal about using both .bash_profile and .bashrc. Will configuring pyenv in this way cause problems for me in the future?

TLDR:
1. I understand the issue with putting both lines in .bashrc is this: #1649.
2. What is the issue with putting both lines in .bash_profile?

**Comment by native-api** [comment]
https://github.com/pyenv/pyenv#basic-github-checkout:

> Add pyenv into your shell <...> to enable autocompletion and all subcommands.

You'll lose this functionality in non-login shells. This includes `pyenv shell` and, if you use Pyenv-Virtualenv, `pyenv activate`.

**Comment by slakshnarasimhan** [comment]
Had to manually edit crewai_tools/tools/selenium_scraping_tool/selenium_scraping_tool.py and change
self.driver = webdriver.Chrome()
to
